container/dockershell.sh

```diff
@@ -1,32 +1,55 @@
-#!/bin/bash -x
+#!/bin/bash
 
 # path within container
 SETUP_PATH="/root/db2eventstore-IoT-Analytics/container/setup"
 USER_VOLUME=${HOME}/eventstore_demo_volume
 
+# Deployment type constants, default to cp4d
+declare -r deployTypeCp4d="cp4d"
+declare -r deployTypeDsx="dsx"
+declare -r deployTypeStandalone="standalone"
+declare -r defaultDeployType=$deployTypeCp4d
+
 if [ -f ${HOME}/.user_info ]; then
-    echo "File: '.user-info' found in the current directory."
-    echo "Extracting user information from the file."
+    printf "File: '.user-info' found in the current directory."
+    printf "Extracting user information from the file."
     EVENT_USER=$(grep  "username" ${HOME}/.user_info |awk {'print $2'})
     EVENT_PASSWORD=$(grep  "password" ${HOME}/.user_info |awk {'print $2'})
 fi
 
 function usage()
 {
 cat <<-USAGE #| fmt
+
 Description:
 This script is the entrypoint of the eventstore_demo container. The script takes
-target Event Store server's public IP, Watson Studio Local's username, and password.
-The script will start docker container in interactive mode using the 
-image: eventstore_demo:latest.
+target Event Store server's public IP, optionally the target Event Store server's REST
+endpoint (required if it differs from the public IP), optionally the target Event Store
+server's deployment type, and the deployment's user and password.
 
+The script will start docker container in interactive mode using the:
+image: eventstore_demo:latest.
 -----------
 Usage: $0 [OPTIONS] [arg]
 OPTIONS:
 ========
---IP        Public IP address of the target Event Store server.
---user      User name of the Watson Studio Local user
---password  Password of the Watson Studio Local user
+--IP        Public IP address or DNS name of the target Event Store server
```

Comment:
do we make any assumptions about the ports ? Maybe here we should specify that this is used for establishing the remote connection to the Db2 Event Store instance, for example, using the setConnectionEndpoints API

Reply:
So as it stands, for both WSL and CP4d, we are using the HostPorts exposed by the engine pods ... we could also use the NodePort exposed by the engine service, but the applications doesn't atm.

But I think we should change this ... in the UI, for the scala/jdbc endpoint we list NodePort ... so that's what you'd use if you're writing an application, yet our samples don't do this ...

Anyway, for now I'll indicate above it's used for our SDKs to connect to the database.